lms/djangoapps/homepage/custom_context_processors.py

```diff
@@ -1,31 +1,15 @@
 from edx_notifications.server.web.utils import get_notifications_widget_context
 
+from .constants import CONFIGS
+
 
 def notifications_configs(request):
```

Comment:
It would be great if we also rename the method to `notifications_context` here and in `lms/envs/common.py` 

Feel free to ignore the suggestions as these aren't exactly in the scope of this ticket. But yes, it can be made if you feel comfortable.